src/Apps/Artist/Routes/WorksForSale/Components/ArtistWorksForSaleEmpty.tsx

```diff
@@ -47,10 +41,11 @@ const ArtistWorksForSaleEmpty: FC<ArtistWorksForSaleEmptyProps> = ({
           <Spacer y={2} />
 
           {newAlertModalEnabled ? (
-            <>
-              <CreateAlertButton onClick={showAlert} />
-              {alertComponent}
-            </>
+            <ArtworkFilterAlertContextProvider
+              initialCriteria={{ artistIDs: [artist.internalID] }}
```

Comment:
Initial criteria including only `artistIDs` but this is refined via the artwork grid filter store further down the stack.